### DISCUSSION TITLE
Next ESMValTool Workshop

### DISCUSSION THREAD

**Discussion by malininae** [ESMValGroup/Community#133] [General]
Hi folks, 

So, since there was a discussion at the monthly meeting, I wanted to raise this question again: when and where the next workshop will take place. I (or someone from my institute) would like to attend, but if the workshop is happening before the 1st of April, our management would need to know by the end of the next week for the financials. @rswamina @valeriupredoi @ehogan @hb326 @ESMValGroup/userengagementteam any news on that? Thanks in advance!

**Comment by alistairsellar** [comment]
Hi @malininae. I'm afraid discussions are still in progress so we won't have a date / place on that timescale. 

If it was after 1st April, would you (or colleague) be thinking of attending *in person*? That would be exciting!

**Comment by valeriupredoi** [comment]
Dunno - I'll go by the statement: it can be anywhere, as long as it's in Barcelona 😁

**Comment by malininae** [comment]
@alistairsellar Thanks for letting me know! Yes, if the budget won't be drastically cut, the plan so far would be to attend in-person :-) I tried to join a few workshops online, but since we are on the Pacific time, it's hard to participate in the discussions or in the coding sessions in the middle of the night.

**Comment by bouweandela** [comment]
I moved the discussion to the community repository to make it more visible to the wider community.

**Comment by malininae** [comment]
If suggestions are being taken, there is a CMIP7 model benchmarking task team meeting in DLR from 21st to 24th of May 2024. If the workshop would be a week before (e.g. May 13th - 17th) or week after (e.g. May 27th-31st), that would be amazing from my perspective, because I would be able to go to Europe once :smiley: That being said, I fully understand that that is my request and others might have a different opinions/needs.

**Comment by hb326** [comment]
Hey all @ESMValGroup/esmvaltool-developmentteam,
I have created a poll for possible dates for the next ESMValTool workshop. We have not decided on a location yet, but wanted to make sure that everybody can block the week/days after we decide when the workshop will be held.

The next ESMValTool workshop will be held (in hybrid format) in late spring/early summer next year. To make sure we can all fit it in our busy calendars we wanted to start the poll for possible workshop dates.
Please indicate which weeks would work for you for the workshop. Most likely the workshop will be three days, but depending on how much we want to discuss and work on, we could also extend it up to altogether five days.
(For some dates only three days are available since these are weeks with German national holidays on the respective Thursdays. If we have the workshop at DLR we would not be allowed to be on campus. If we have the workshop somewhere else, we could use the full week.)
Thanks! 

https://terminplaner6.dfn.de/en/b/c4458d6a0286eab9e8f63e32f362ea48-496371

**Reply by malininae** [reply]
@hb326 we supposed to choose 'book' for the dates we think we could attend, correct? No to leave the comments?

**Reply by rbeucher** [reply]
I'll be travelling to Europe to attend the EGU meeting and should be able to join a workshop if it happens close to the EGU dates.

**Reply by hb326** [reply]
Ok, I have tried to change the poll system. The one I used (DFN) is not really nice to work with, and not easy to understand.
I tried to use GitHub polls, but it only lets you choose one option. So the not so nice to work with option it is...

**Reply by hb326** [reply]
@malininae: yes, just click "book" for the dates that would work for you, and do not click "book" for the dates that would not work for you. It should be possible to leave comments with the poll. Let me know if that is not the case!

**Reply by valeriupredoi** [reply]
so I have STORNO against all possible dates now (coz I can attend them all). Just making sure that's how it's supposed to look like when one accepts all possible available dates, STORNO is a bit of an obtuse word to me :grin:
![STORNO](https://github.com/ESMValGroup/Community/assets/28983971/eb6d76a5-a0b8-4665-b12b-ff3174a83acb)

**Reply by valeriupredoi** [reply]
motion to change STORNO with SCHNITZEL

**Reply by hb326** [reply]
Well done @valeriupredoi! You did fill out the poll correctly! That's what it is supposed to look like when you "booked" all time slots. Not sure I can make it happen that it would show "SCHNITZEL" instead of "STORNO", but you know, for you, I will try my best.  :)

**Comment by hb326** [comment]
Hey @ESMValGroup/esmvaltool-developmentteam, 
just a gentle reminder to you all to fill out the poll to find the next workshop dates if you have not done so already, so that we can fix the dates soon.
Thanks!

**Comment by hb326** [comment]
Hey @ESMValGroup/esmvaltool-developmentteam,
one last nudge for you to remind you to fill out the poll to find the next workshop dates. If you have not filled out the poll yet, please do so by the end of **Wednesday, Dec 20th**. 
Thanks!

**Reply by diegokam** [reply]
Thanks for the reminder! I have just filled the poll. All dates sound good to me! Thanks!

**Comment by hb326** [comment]
Ok, the poll is closed now. We have a date!
